### ISSUE TITLE
[Creator Starter Kit] BUG: icon default tooltip is not helpful, would be better to not have one

### ISSUE THREAD

**Issue by Jenefer-Monroe** [microsoft/powercat-creator-kit#221]
**Describe the bug**
Today when you use the icon component you get these tooltips by default. These all need fixed up
![image](https://user-images.githubusercontent.com/89584709/207628526-f76ea136-5a5b-42eb-b762-d08f75d80a3c.png)

**To Reproduce**
Use a tooltip component, view in player, hover over icon --> see "Icon component"

**Expected behavior**
I would expect not to see a tooltip by default. As is, this just leads the makers to have to go do more work when perhaps they didnt need a tooltip at all. 

**Screenshots**
![image](https://user-images.githubusercontent.com/89584709/207628526-f76ea136-5a5b-42eb-b762-d08f75d80a3c.png)

**Additional context**
None

**Comment by denisem-msft** [comment]
Thanks for the feedback @Jenefer-Monroe. This is consistent with how the native canvas/custom page controls are designed and your point on how it causes additional work on the maker makes sense. We'll add this to the backlog for next release.

For now, can you please try removing the value in the tooltip?

**Comment by Jenefer-Monroe** [comment]
Yes absolutely, not urgent, just what I like to call a "papercut" :)
Thank you for considering it!
